img2dataset/downloader.py

```diff
@@ -348,26 +401,63 @@ def download_one_file(url_list):
             timeout=timeout,
         )
 
+        # Start a W&B run
+        run = wandb.init(
+            project="img2dataset", config=config_parameters, anonymous="allow"
```

Comment:
I think it could be good to add an option to let the user choose the name of the wandb project